### DISCUSSION TITLE
files inside repo are not seen on bitbucket

### DISCUSSION THREAD

**Discussion by Santhosh105** [community/community#37433] [Other Feature Feedback, Questions, & Ideas]
Hi All,

I was trying to git push a repo which was successful

But i cannot see the files on remote repo

Both are on master branch.

**Comment by ossanna16** [comment]
Hi @Santhosh105,
Thanks for being a part of the GitHub Community! We're glad you're here.

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Bitbucket. It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
